testing/functional/tests/dataGrid/focusedRow.ts

```diff
@@ -496,13 +496,13 @@ test('Focused row should not fire onFocusedRowChanging, onFocusedRowChanged even
     await t.expect(ClientFunction(() => (window as any).focusedRowChanged_Counter)()).eql(1);
 }).before(() => createWidget('dxDataGrid', () => {
     const data = function() {
-        const data = [];
+        const result = [];
 
-        for(let i = 0; i < 200; i++) {
-            data.push({ id: i, c0: 'c0', c1: `c1_${i % 20}` });
+        for(let i = 0; i < 200; i += 1) {
```

Comment:
Why is ++ changed to += 1 everywhere?

Reply:
Because the unary ++ and -- operators are subject to automatic semicolon insertion, differences in whitespace can change semantics of source code.

Reply:
It is default airbnb rule